api/swagger.yaml

```diff
@@ -1939,6 +1939,39 @@ definitions:
             description: "A list of additional groups that the container process will run as."
             items:
               type: "string"
+          Privileges:
+            type: "object"
+            description: "Security options for the container"
+            properties:
+              CredentialSpec:
+                type: "object"
+                description: "CredentialSpec for managed service account (Windows only)"
+                properties:
+                  File:
+                    type: "string"
+                    description: "Load credential spec from this file"
+                  Registry:
+                    type: "string"
+                    description: "Load credential spec from this registry key"
```

Comment:
Technically it is a registry _value_ within the CredentialSpecs registry key, so I would change this to 'registry value'. Comparing to a file system, a key is roughly equivalent to a directory, and a value, a file.